app/src/main/java/com/example/oddhours/data/repository/JobRepository.kt

```diff
@@ -16,15 +33,19 @@ class JobRepository() {
      */
     var shiftsList: List<ShiftsModel>? = null
 
+//    var context: Context? = null
+
     /**
      * TODO: this method should make use of the return object / values from DB (instead of taking in jobName and jobLocation)
      * and create all the jobs then probably store it in jobModelList for the current app session. this should be called from
      * MainActivity after we connect to the DB and query it to get all current jobs the user has stored. the return list is
      * what we feed into the recyclerview adapter!
      */
-    fun buildJobList(jobName: String, jobLocation: String): JobModel {
+    fun buildJobList(): List<JobModel> {
         Log.i(TAG, "buildJobList() started")
-        return JobModel(1, jobName, jobLocation, getTotalHoursForWeek(jobName))
+        var db = DatabaseHelper(context!!)
```

Comment:
can probably have it like so: `val db = DatabaseHelper(context)` since db isn't being changed anywhere and we already know context is not null